vmad.egg-info/top_level.txt

```diff
@@ -0,0 +1 @@
+vmad
```

Comment:
Hmm. Looks like you added the vmad.egg-info directory -- it doesn't belong to the source code. Please remove it with git rm.

Reply:
Done, the full directory is removed. Sorry about that.